clean_project/clean_file_test.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+import tempfile
+import unittest
+
+
+from clean_file import clean_up, sanitize
+
+
+class CleanFileTest(unittest.TestCase):
+
+    def test_punctuation_begin_sentence(self):
+        string_test = u"-»¿;hi how are you"
+        self.assertEqual("-hi how are you", sanitize(string_test))
+
+    def test_punctuation_end_sentence(self):
+        string_test = u"hi how are you???»¿;"
+        self.assertEqual("hi how are you", sanitize(string_test))
+
+    def test_punctuation_middle_sentence(self):
+        string_test = u"hi, how,»¿; are, you"
```

Comment:
it would be good to test also other characters rather than these ones only, what do you say